src/game/Handlers/CharacterHandler.cpp

```diff
@@ -1071,9 +1071,26 @@ void WorldSession::HandlePlayerLoginFromDB(LoginQueryHolder* holder)
         pCurrChar->SendTalentsInfoData(false);              // original talents send already in to SendInitialPacketsBeforeAddToMap, resend reset state
         SendNotification(LANG_RESET_TALENTS);
     }
-
-    if (pCurrChar->HasAtLoginFlag(AT_LOGIN_FIRST))
-        pCurrChar->RemoveAtLoginFlag(AT_LOGIN_FIRST);
+    
+    // Fix done by mthsena;
```

Comment:
we do not add credits using comments in the code, the credits to the author are added automatically thanks to the versioning system